src/htrun/host_tests_runner/target_base.py

```diff
@@ -22,19 +22,17 @@
 from ..host_tests_logger import HtrunLogger
 
 
-class Mbed:
+class Base:
```

Comment:
Ah, here is the enigmatic `Base`. Can we call it `BaseHostTest` instead?

Reply:
I thought to use BaseHostTest but already `greentea/src/htrun/host_tests/base_host_test.py` has the same class name, I will see what's the meaningful name

Reply:
Ah, `BaseHostTest` is already taken by another class. Maybe `TargetBase` or `DeviceBase` then, as this object seems to be concerned with managing devices.

Reply:
we will go with `TargetBase`